csharp/ql/lib/CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
+## 0.0.10
+
 ## 0.0.9
 
 ### Major Analysis Improvements
 
-* Added support for C# 10 lambda improvements
-  * Explicit return types on lambda expressions.
-  * Lambda expression can be tagged with method and return value attributes.
+Added support for C# 10 lambda improvements
+* Explicit return types on lambda expressions.
+* Lambda expression can be tagged with method and return value attributes.
```

Comment:
How would I need to fix this so that I wouldn't have to do it in every release?
This has been fixed in 2.8.1: https://github.com/github/codeql/pull/7975/commits/c5d917eb72d9eb93af12577496e87a26d5c06852

Reply:
I guess you should have updated `csharp/ql/lib/change-notes/released/0.0.9.md` too.

Reply:
I suspect this file is (re-)generated by concatenating all `sharp/ql/lib/change-notes/released/*.md` files.

Reply:
Thanks. I hope this fixes it.  https://github.com/github/codeql/pull/8228